documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: documentation
+menu: check
+subtitle: 'Tutorial: Using Check with SQL Server'
+---
+# Tutorial: Using Flyway Check with SQL Server
+
+{% include enterprise.html %}
+
+## Introduction
+
+This is an enteprise feature, but the beta version is available to Teams users.
+
+The new `check` command is run prior to migrating, to give us confidence in the state of the post-migration schema by producing a report. 
+
+One or more of the following flags must be set, which determine what the report contains:
+
+ - `-changes` produces a report of all the changes that will be applied to the schema in the next migration. 
+ - `-drift` produces a report showing objects in the schema which are not the result of any of the currently applied migrations, i.e. previous changes made outside of Flyway.
+
+More information can be found [here](/documentation/command/check).
+
+## Download the latest Beta version of Flyway Enterprise CLI
+
+First, [download the latest beta version of the Flyway Enterprise CLI](/documentation/learnmore/beta) and extract its contents. 
+
+## Setting up databases
+
+To use `check` we will need access to two databases:
+
+- The **target** database. This is the database we're interested in making changes to. 
+- A **temporary** database. Flyway will clean this database, apply migrations to it, and make comparisons in order to generate **change and drift reports**. 
+
+In this example, there are two databases in our SQL Server instance: `foobar` (the target) and `check_temp_db`. Both can be accessed by the `sa` user.
+
+## Configuration
```

Comment:
Since the single report PR was merged, `outputLocation` was replaced with `reportFilename` which is a required parameter

The corresponding docs PR is [here](https://github.com/flyway/flywaydb.org/pull/443)

The tutorial will need updating to include this new parameter since this change has already been merged (albeit not released)